### PR TITLE
GEF: Companies House and OS Places integration

### PR DESCRIPTION
**Pull request by mwyld**

### CODE REVIEW

File: reference-data-proxy/package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "start:dev": "npx nodemon src/index.js",
     "unit-test": "jest --notify --coverage --verbose --config=unit.jest.config.js ",
     "unit-test-quick": "jest --config=unit.jest.config.js --testTimeout=10000",
-    "api-test": "jest --notify --coverage --verbose --runInBand --config=api-test.jest.config.js ",
+    "api-test": "jest --notify --coverage --verbose --runInBand --config=api-test.jest.config.js --watch",
```
Comment (pricea1) [inline reference-data-proxy/package.json:12]: Can you remove --watch. The api-test is run as part of the pipeline which I don't think will finish & close with --watch flag

**Review by pricea1 — COMMENTED** [review]
Just a jest flag to remove I think otherwise LGTM

File: reference-data-proxy/package.json (hunk 1 above)
Comment (mwyld) [reply reference-data-proxy/package.json]: removed

**Review by pricea1 — APPROVED** [review]
Good to go

**Pull request merged**